code/global.dm

```diff
@@ -783,3 +784,24 @@ var/global/mentorhelp_text_color = "#CC0066"
 		globalImages[key] = null
 		globalImages.Remove(key)
 	return
+
+/proc/addAIImage(var/image/I, var/key)
+	if(I && length(key))
+		aiImages[key] = I
+		for(var/mob/M in AIs)
+			if (M.client)
+				M << I
+		return I
+	return
```

Comment:
return null?

Reply:
null here serves as a way to tell if the 'add' failed

Reply:
```suggestion
	return null
```